backend/benefit/messages/automatic_messages.py

```diff
@@ -102,3 +84,26 @@ def notify_applicant_by_email_about_new_message(application):
                 "error",
             )
             return 0
+
+
+def send_application_reopened_message(
+    user, application, additional_information_needed_by
+):
+    """
+    :param user: The handler who is setting the application to ADDITIONAL_INFORMATION_REQUESTED status
+    :param application: The application being reopened
+    :param application: The last response date
```

Comment:
`application` two times? Anyway, I would write comment only if the same information could not be told by code itself; by more descriptive naming, type hints, etc. Comments can always be wrong and outdates quickly.